modules/kuberay-cluster/main.tf

```diff
@@ -105,44 +105,26 @@ resource "kubernetes_network_policy" "kuberay-head-network-policy" {
     }
 
     ingress {
-      # Ray Client server
```

Comment:
In this case, will it break out README to submit the job using port forward?

Reply:
`kubectl port-forward` always works with networkpolicy